build/yamls/antrea-windows-ovs-containerd.yml

```diff
@@ -0,0 +1,361 @@
+      # then AntreaProxy will only handle Services with the label that equals the provided value. If it is not set,
+      # then AntreaProxy will only handle Services without the "service.kubernetes.io/service-proxy-name" label,
+      # but ignore Services with the label no matter what is the value.
+      serviceProxyName: ""
+
+    nodePortLocal:
+    # Enable NodePortLocal, a feature used to make Pods reachable using port forwarding on the host. To
+    # enable this feature, you need to set "enable" to true, and ensure that the NodePortLocal feature
+    # gate is also enabled (which is the default).
+    #  enable: false
+    # Provide the port range used by NodePortLocal. When the NodePortLocal feature is enabled, a port
+    # from that range will be assigned whenever a Pod's container defines a specific port to be exposed
+    # (each container can define a list of ports as pod.spec.containers[].ports), and all Node traffic
+    # directed to that port will be forwarded to the Pod.
+    #  portRange: 40000-41000
+  antrea-cni.conflist: |
+    {
+        "cniVersion":"0.3.0",
+        "name": "antrea",
+        "plugins": [
+            {
+                "type": "antrea",
+                "ipam": {
+                    "type": "host-local"
+                },
+                "capabilities": {"dns": true}
+            }
+        ]
+    }
+kind: ConfigMap
+metadata:
+  labels:
+    app: antrea
+  name: antrea-windows-config-cb2hmgg648
+  namespace: kube-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app: antrea
+    component: antrea-agent
+  name: antrea-agent-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: antrea
+      component: antrea-agent
+  template:
+    metadata:
+      annotations:
+        microsoft.com/hostprocess-inherit-user: "true"
+      labels:
+        app: antrea
+        component: antrea-agent
+    spec:
+      containers:
+      - args:
+        - -file
+        - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/Run-AntreaAgent-OVS-Containerd.ps1
+        command:
+        - powershell
+        env:
+        - name: POD_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.name
+        - name: POD_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: NODE_NAME
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.nodeName
+        image: antrea/antrea-windows:latest
+        imagePullPolicy: IfNotPresent
+        lifecycle:
+          postStart:
+            exec:
+              command:
+              - powershell
+              - -file
+              - $env:CONTAINER_SANDBOX_MOUNT_POINT/var/lib/antrea-windows/PostStart-AntreaAgent-Containerd.ps1
+          preStop:
+            exec:
+              command:
+              - powershell
+              - Disable-VMSwitchExtension -Name 'Open vSwitch Extension' -VMSwitchName
```

Comment:
After discussing with @wenyingd, I agree it's better to introduce one more step to check if antrea-hnsnetwork exists on host before disable VMSwitchExtension. @Atish-iaf Could you add a new script PreStop-AntreaAgent-Containerd.ps1 here to replace the single command. The script should be :
```
  PostStart-AntreaAgent-Containerd.ps1: |
    $networkName = "antrea-hnsnetwork"
    $networks = Get-HnsNetwork
    $networkExists = $false
    foreach ($network in $networks) {
        if ($network.Name -eq $networkName) {
            $networkExists = $true
            break
        }
    }
    if ($networkExists) {
        Disable-VMSwitchExtension -Name "Open vSwitch Extension" -VMSwitchName "antrea-hnsnetwork"
    }
```

Reply:
added.